pephubclient/samples/samples.py

```diff
@@ -0,0 +1,38 @@
+from ..files_manager import FilesManager
+
+
+class Samples:
+    def __init__(self):
+        self.jwt_data = ""
+
+    def get(
+        self,
+        namespace: str,
+        name: str,
+        tag: str,
+        sample_name: str = None,
+    ):
+        ...
+
+    def create(
```

Comment:
would this take a peppy.Project object?

and instead of `create` should this ` add_to_db` or something?